### ISSUE TITLE
[Footer] Social media icons too small to respond consistently when tapped (multiple devices)

### ISSUE THREAD

**Issue by colinpmacarthur** [18F/web-design-standards#346]
Social media icons are too small to respond consistently when tapped:
![footer social link spacing](https://cloud.githubusercontent.com/assets/2092076/9476746/617604d8-4b3c-11e5-97d1-0b765b9d7d59.jpg)

## Affected devices
- Moto E, Android 4.4, Chrome 44
- iPhone 6, iOS 8.4

**Comment by joshbruce** [comment]
The Apple iOS HIG recommends 44x44pt hit area - this can, generally speaking, be converted to 44x44px. Note: The hit area does not need to be the visible size of the link.

For example:

```
a {
  display: inline-block;
  height: 44px;
  width: 44px;
}
```

This doesn't work so well when the contents are variable - true links - but could be a workable solution here.

**Comment by maya** [comment]
We'll need to implement the larger icons in #706 resolve this.

**Comment by Drakenhart** [comment]
Sometimes it may not be the icons, but the spacing around them as well. If the icons are smaller, just make sure there is adequate padding/margins to give it a 44px x44px tap area.
